### ISSUE TITLE
Get Number of Users and Posts

### ISSUE THREAD

**Issue by ryanleesipes** [discourse/discourse_api#101]
I'm making a dashboard that gives up-to-date numbers of users and posts for our Discourse install. How can I pull this information out using this gem?

**Comment by oblakeerickson** [comment]
Currently right now there isn't method inside of this gem that would give you that data, but I'll see if I can whip something up real quick.

I think we can use this discourse endpoint: `http://localhost:3000/admin/dashboard.json?_=1456689376553` to get the data. The discourse admin dashboard calls this endpoint to get all the numbers. 

![screenshot from 2016-02-28 13 12 04](https://cloud.githubusercontent.com/assets/1490496/13381605/1f66f95c-de1d-11e5-9476-1b603e800a20.png)

If we use the data from the 'All' column for 'New Users' and for 'Posts' it should give you what you are looking for.

**Comment by oblakeerickson** [comment]
@ryanleesipes see for an example https://github.com/discourse/discourse_api/blob/master/examples/dashboard.rb

You can now call

```ruby
client.client.get_dashboard_stats_totals
```
which will output:

```
{"users"=>9, "topics"=>230, "posts"=>441}
```

To get what you are looking for.

I published a new version 0.8.0 to ruby gems so be sure to update your gem. 

Please close this issue if this method gets the data you need.

**Comment by ryanleesipes** [comment]
This was all fantastic feedback! As a note to others who read this, you can pull these individual stats simply by putting client.get_dashboard_stats_totals['users'] for instance, to get the number of users. Awesome stuff! Thanks for being so helpful!
